### PR TITLE
DOCS-4429: CSPM Compliance Framework Reports

### PR DESCRIPTION
**Pull request by drichards-87**
### What does this PR do?
Rewrites the CSPM Findings Report article.

### Motivation
DOCS-4429

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/security/cspm/frameworks_and_benchmarks.md [hunk 1]
```diff
@@ -38,31 +37,33 @@ Each [OOTB rule][1] maps to one or more controls within a compliance standard or
...
 
-**Note**: Datadog CSPM provides you with visibility into whether your resources are configured in accordance with certain detection rules. Datadog's OOTB detection rules address various regulatory frameworks, benchmarks, and standards (“Security Posture Frameworks”). Datadog CSPM does not provide an assessment of your actual compliance with any Security Posture Framework, and the OOTB rules may not address all configuration settings that are relevant to the Security Posture Frameworks. To be clear, just because your resources pass the OOTB Rules does not mean that you are meeting all the requirements under any particular Security Posture Framework. Datadog is not providing legal or compliance advice or guidance, and it is recommended that you use Datadog CSPM in consultation with your legal counsel or compliance experts.
+Datadog also provides Datadog Security Labs, a set of recommendations developed by Datadog internal security experts. Based on common cloud security risks we have observed at Datadog, this ruleset aims to help users new to cloud security easily remediate high-impact misconfigurations across their cloud environments.
```
Comment (hdelaby) [inline content/en/security/cspm/frameworks_and_benchmarks.md:40]: Datadog Security Labs has been renamed `Essential Cloud Security Controls`

**Review by hdelaby — CHANGES_REQUESTED** [review]
Small nit, good otherwise!

File: content/en/security/cspm/frameworks_and_benchmarks.md (hunk 1 above)
Comment (drichards-87) [reply content/en/security/cspm/frameworks_and_benchmarks.md:40]: Thanks for clarifying. I'll fix it in the PR.

File: content/en/security/cspm/frameworks_and_benchmarks.md [hunk 2]
```diff
@@ -38,31 +37,33 @@ Each [OOTB rule][1] maps to one or more controls within a compliance standard or
...
-- [Microsoft Teams][18]
-- [Webhooks][19]
-- Email
+{{< img src="security/cspm/frameworks_and_benchmarks/report-2.png" alt="Set a findings time window using the dropdown" style="width:100%;">}}
 
-In the “Notify” section, configure zero or more notification targets for each rule case. You cannot edit the preset severity.
+Select a rule to view details about the misconfigured resources, the rule description, its framework or industry benchmark mapping, and suggested remediation steps.
 
-{{< img src="security/cspm/frameworks_and_benchmarks/notification.png" alt="Select a severity and notification target" >}}
+{{< img src="security/cspm/frameworks_and_benchmarks/failed-finding.png" alt="Set a findings time window using the dropdown" style="width:75%;">}}
```
Comment (estherk15) [inline content/en/security/cspm/frameworks_and_benchmarks.md:66]: I think the alt description is for a different image?

File: config/_default/menus/menus.en.yaml [hunk 3]
```diff
@@ -2609,20 +2609,22 @@ main:
     parent: cspm
     identifier: cspm_schema_gcp_test
     weight: 104
+  - name: Compliance Reports
+    weight: 105
```
Comment (estherk15) [inline config/_default/menus/menus.en.yaml:2613]: Are you leaving the URL as is? Also, are you leaving the Findings and Reports as a placeholder?

**Review by estherk15 — CHANGES_REQUESTED** [review]

**Review by hdelaby — APPROVED** [review]

**Review by estherk15 — APPROVED** [review]

**Pull request merged**